### PR TITLE
Sf cmake intergration

### PR DESCRIPTION
**Pull request by SvenStruanFinlay**
Hmm, this is going ot be pushing the last PR for CMAKE into the dev branch.

### CODE REVIEW

File: Intel_SighPP_CMake/CMakeSettings.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        {
+          "name": "spdlog_DIR",
+          "value": "C:/Users/svens/vcpkg/installed/x64-windows/share/spdlog",
+          "type": "PATH"
+        }
+      ],
+      "cmakeToolchain": "C:/Users/svens/vcpkg/scripts/buildsystems/vcpkg.cmake"
+    }
+  ]
+}
```
Comment (SvenStruanFinlay) [inline Intel_SighPP_CMake/CMakeSettings.json:53]: I think there was a comment on this part. These cmake paths can be overwritten on new cache creation.

**Review by SvenStruanFinlay — COMMENTED** [review]
So Apparently I like random spacing. Will fix while I integrate object tracking!

File: Intel_SighPP_CMake/CMakeSettings.json (hunk 1 above)
Comment (GisliGeorgs) [reply Intel_SighPP_CMake/CMakeSettings.json:53]: Yeah, I think ideally we'd have a placeholder value.

File: Intel_SighPP_CMake/Intel_SighPP_CMake/cv-helpers.hpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```
Comment (GisliGeorgs) [inline Intel_SighPP_CMake/Intel_SighPP_CMake/cv-helpers.hpp:1]: We need to add Licences, I can do it soon 😄

File: README.md [hunk 3]
```diff
@@ -1,12 +1,71 @@
...
+## Core sources
 
-## Packages required
+## Installation and Development
+### Packages required
 This project uses [Vcpkg](https://github.com/Microsoft/vcpkg) for library dependencies. Install the following libraries with Vcpkg to run the solution (make sure to install the 64-bit versions, as this system is being developed with 64-bit systems in mind):
 
 * realsense2
-* opencv3
+* opencv3[contrib]
```
Comment (GisliGeorgs) [inline README.md:12]: Should it just be contrib?

**Review by GisliGeorgs — APPROVED** [review]
Should .DS_STORE files be added or should they be ignored with gitignore?
I'm not sure exactly what is new 😄 But yeah. Looks good.

**Comment by lspcool** [comment]
Looks good. a lot of files

**Review by lspcool — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (SvenStruanFinlay) [reply README.md:12]: Oh ye probs should be both

**Pull request merged**